AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Images-out-of-focus.rst

```diff
@@ -0,0 +1,117 @@
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#########################
+AuxTel Image out of focus
+#########################
+
+.. _Image-out-of-focus-Procedure-Overview:
+
+Overview
+========
+
+This procedure should be used when images appear out of focus due to a *WEP failure* or other related issues. Users will notice images ranging from slightly out of focus to large donut appearances when hexapod offsets are not correctly applied.
+
+.. _AuxTel-Image-out-of-focus-Procedure-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+- The focus script :file:`latiss_wep_align` might fail, leaving the system out of focus.
+- These out of focus images might vary from slightly out of focus image when the WEP script took the system out of focus,
+  
+.. figure:: ./_static/min-focus.png
+    :name: in-focus
+    :width: 400
+
+    Slightly out-of-focus image
+
+to complete out-of-focus images that can appear as large donuts when hexapod offsets are left in intra or extra focus positions due to script failure.
+
+.. figure:: ./_static/out-focus.png
+    :name: out-focus
+    :width: 400
+
+    Wildly out-of-focus image
+
+.. _AuxTel-Image-out-of-focus-Procedure-Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. _AuxTel-Image-out-of-focus-Procedure-Critical-Step-1:
+
+#. Pause ATScriptQueue.
+
+#. Clear all ATAOS offsets for the x, y, and z axes. Load and run the standard script :file:`auxtel/offset_ataos.py` **moving the script up in the queue** with the following configuration.
+    
+     .. code-block:: python
+        :caption: auxtel/offset_ataos.py
+
+        reset_offsets: all
+
+#. Run the :file:`auxtel/latiss_wep_align.py` external script with the configuration below to find a new target in the area of the sky of your choice. Replace `az` and `el`.
+
+     .. code-block:: python
+        :caption: auxtel/latiss_wep_align.py
+
+        find_target:
+           az: 135
+           el: 40
+           mag_limit: 8.0
+
+
+#. Keep playing the rest of the scripts in the queue, ensuring the system is now in focus.
+
+#. Complete the procedure by verifying the system is in focus.
+
+.. _AuxTel-Image-out-of-focus-Procedure-Post-Condition:
+
+Post-Condition
+==============
+
+- System is back in focus.
+
+.. _AuxTel-Image-out-of-focus-Procedure-Contingency:
+
+Contingency
+===========
+
+If the procedure was not successful, report the issue immediately in the `#summit-auxtel <https://lsstc.slack.com/archives/C01K4M6R4AH>`__ channel.
+
+- If step 2 doesn't work and you see a large number in the *ATAOS offsets values*, it is because offsets had been accumulated. 
+  
+  Restart *ATAOS*: Cycle *ATAOS* to ``STANDBY`` and back to ``ENABLED`` state. 
```

Comment:
1. *ATAOS* --- > :guilabel:`ATAOS`
2. Mention that we do this from *ASummary State* tab in LOVE

Reply:
The CSC don't have an explicit standard format currently (but they should), but we have been using italics to highlight lightly. So, I have converted all CSC mentions to italics.

Reply:
I will keep this in mind for my docs as well!